crates/ctranslate2-bindings/build.rs

```diff
@@ -54,26 +54,28 @@ fn link_static() -> PathBuf {
             .define("WITH_MKL", "OFF")
             .define("OPENMP_RUNTIME", "NONE");
 
-        if cfg!(target_arch = "aarch64") && check_jetson() {
-            config
-               .define("WITH_CUDA", "ON")
-               .define("WITH_CUDNN", "ON")
-               .cxxflag("-mcpu=native")
-        } else if cfg!(target_feature = "sse4.1") {
-            config
-                .define("WITH_OPENBLAS", "ON")
-                .cxxflag("-msse4.1")
+        if cfg!(target_feature = "sse4.1") {
+            config.cxxflag("-msse4.1");
+        }
+        
+        if cfg!(feature = "link_static_cuda") {
+            config.define("WITH_CUDA", "ON");
+            
+            if cfg!(target_arch = "aarch64") && check_jetson() {
+                config
+                   .define("WITH_CUDNN", "ON")
```

Comment:
this should be enabled no matter if aarch64 / x86?

Reply:
I found on Jetson cuDNN comes with JetPack by default. But on Linux (e.g., Ubuntu), cuDNN needs to be installed separately with CUDA. Does Tabby use cuDNN? If not, I'll remove it

Reply:
Should be good to align with ctranslate2's CUDA compilation configuration

https://github.com/OpenNMT/CTranslate2/blob/master/python/tools/prepare_build_environment_linux.sh#L56C1-L57C1

Reply:
OK, I got it.

Reply:
@wsxiaoys `CUDA_DYNAMIC_LOADING=ON` in the script given by you probably conflicts with the literal meaning of `link_static_cuda`

Reply:
I was specifically mentioning the `WITH_CUDNN=ON` in ctranslate2's cuda config

Reply:
@wsxiaoys How about this version. I've deleted unused code.